### DISCUSSION TITLE
Equipo hackaton

### DISCUSSION THREAD

**Discussion by DavidSantana872** [Launch-X-Latam/MisionFrontEnd#560] [Team Posts]
Un espacio en algún grupo, alguien que me adopte en su equipo 😔

**Comment by Perotindev** [comment]
Nosotros somos dos si querés sumarte

**Comment by ArnedoAn** [comment]
Hola, tienen espacio para otro?

**Comment by JoseEduardoVelazquezCN** [comment]
Puedo unirme ¿? Mi correo es ***@***.***

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Perotindev** [comment]
No hay problema, si nos unimos todos ya somos 5, si confirman todos

**Comment by Perotindev** [comment]
Pasen sus mails de innova, el mío es lx13362@innovaccion.mx

**Comment by ArnedoAn** [comment]
Mi correo es: lx08609@innovaccion.mx

**Comment by DavidSantana872** [comment]
Listo, mi correo: lx07020.@innovacion.mx

**Comment by DavidSantana872** [comment]
@JoseEduardoVelazquezCN  tu correo de innovacion es?

**Comment by Perotindev** [comment]
Listo esperamos q contesté José ya somos 5

**Comment by JoseEduardoVelazquezCN** [comment]
***@***.*** este es mi correo !!!

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Perotindev** [comment]
Aparece oculto amigo!

**Comment by JoseEduardoVelazquezCN** [comment]
***@***.*** en lo que modifico el otro

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by DavidSantana872** [comment]
@JoseEduardoVelazquezCN  jajaja de nuevo salio oculto

**Comment by JoseEduardoVelazquezCN** [comment]
l x 0 4 6 5 4 @ i n n o v a c c i o n . m x así si ¿? Xd

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Perotindev** [comment]
Ahí va, perfecto entonces, ya somos 5

**Comment by Max80318** [comment]
Alguien que no tenga grupo que me una a uno
***@***.***

**Comment by argeyx** [comment]
Tampoco tengo
